File: Hoops.Core/Entities/Director.cs
using System;
using System.Collections.Generic;

namespace Hoops.Core.Entities
{
    public partial class Director
    {
        public int DirectorId { get; set; }
        public int? CompanyId { get; set; }
        public int? PersonId { get; set; }
        public int? Seq { get; set; }
        public string Title { get; set; }
        public byte[] Photo { get; set; }
        public string PhonePref { get; set; }
        public int? EmailPref { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string CreatedUser { get; set; }
    }
}

File: hoops.ui/src/app/domain/director.ts
export class Director {
  directorId!: number;
  companyId?: number;
  personId!: number;
  seq?: number;
  title!: string;
  name?: string; // From VwDirector, not in base Director model
  firstName?: string;
  lastName?: string;
  phone?: string;
  cellPhone?: string;
  workPhone?: string;
  email?: string;
  address1?: string;
  city?: string;
  state?: string;
  zip?: string;
  phonePref?: string;
  emailPref?: number;
  createdDate?: Date;
  createdUser?: string;
  photo?: any;
  constructor(
    directorId: number,
    name: string,
    companyId: number,
    personId: number,
    seq: number,
    title: string
  ) {}
}

// Interface for director list display
export interface DirectorListItem {
  id: number; // For BaseList compatibility
  directorId: number;
  name: string;
  title: string;
  seq: number;
}
